### ISSUE TITLE
[BUG] FutureWarning from fillna call

### ISSUE THREAD

**Issue by pamelafox** [microsoft/promptflow#2702]
**Describe the bug**

When I use the azure-ai-generative SDK's evaluate function(), it calls PromptFlow code which results in this warning in my logs:

/Users/pamelafox/ai-rag-chat-evaluator/.venv/lib/python3.11/site-packages/promptflow/_sdk/operations/_local_storage_operations.py:516: FutureWarning: Setting an item of incompatible dtype is deprecated and will raise an error in a future version of pandas. Value '(Failed)' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.
  outputs.fillna(value="(Failed)", inplace=True)  # replace nan with explicit prompt

I am on promptflow 1.6 due to an issue with 1.7 and evaluate SDK, but I don't know if this warning is fixed yet, as the code seems to be similar in main.

**How To Reproduce the bug**

I experience it when I'm using https://github.com/Azure-Samples/ai-rag-chat-evaluator, which calls the evaluate function from azure-ai-generative, which uses promptflow behind the scenes.

**Expected behavior**

I expect no warnings.

**Comment by zhengfeiwang** [comment]
ack, i will work on this to fix the warning

**Comment by pamelafox** [comment]
I dont think you need more of my input. I havent checked to see if this is fixed in new promptflow-evals.

**Comment by pamelafox** [comment]
I already replied.
